src/foam/u2/view/ChoiceView.js

```diff
@@ -60,7 +60,7 @@ foam.CLASS({
 
         try {
           if ( ! n && this.placeholder ) {
-            this.data  = undefined;
+            this.data  = this.defaultValue !== undefined ? this.defaultValue : undefined;
```

Comment:
Ha, I guess this could just be 
```suggestion
            this.data  = this.defaultValue;
```